test/api/test_cpu.cpp

```diff
@@ -64,21 +64,22 @@ TEST_CASE(zero_parameter)
     CHECK(bool{shapes_before.front() == outputs.front().get_shape()});
 }
 
-TEST_CASE(set_scalar_parameter)
+TEST_CASE(scalar_shape)
 {
-    auto p1 = migraphx::parse_onnx("add_bcast_test.onnx");
-    migraphx::shape s1(migraphx_shape_float_type, {3, 4});
-    auto param_shapes = p1.get_parameter_shapes();
-    auto s1_orig      = param_shapes["1"];
-    CHECK(bool{s1 == s1_orig});
+    auto s = migraphx::shape(migraphx_shape_float_type);
+    EXPECT(s.lengths().size() == 1);
+    EXPECT(s.strides().size() == 1);
+    EXPECT(s.lengths().front() == 1);
+    EXPECT(s.strides().front() == 0);
+}
 
-    migraphx::onnx_options option;
-    option.set_input_parameter_shape("1", {});
```

Comment:
This line will throw an exception and this is what we meet in ORT

Reply:
I added this test back in and I fixed the null pointer error.